packages/api/bin/serveUtils.js

```diff
@@ -89,7 +119,7 @@ async function addGranules(granules) {
       );
       await indexer.indexGranule(es.client, newGranule, es.index);
       const dbRecord = await translateApiGranuleToPostgresGranule({
-        newGranule,
+        dynamoRecord: newGranule,
```

Comment:
Eventually we should rename this argument (not in this PR)

Reply:
Also looks like this was actually broken 😢

Reply:
I'm not entirely sure how that wound up still being dynamoRecord.  I'm wondering if there's a merge issue 🤔

Reply:
Not relevant to this PR but yeah, it's possible. There may have been an update to the key names in main and a separate one in the feature branch.